client/post-editor/index.js

```diff
@@ -14,6 +14,7 @@ module.exports = function() {
 	page( '/post', controller.pressThis, sitesController.siteSelection, sitesController.sites );
 	page( '/post/new', () => page.redirect( '/post' ) ); // redirect from beep-beep-boop
 	page( '/post/:site?/:post?', sitesController.siteSelection, sitesController.fetchJetpackSettings, controller.post );
+	page( '/post/revision/:site?/:post?', sitesController.siteSelection, sitesController.fetchJetpackSettings, controller.revision );
```

Comment:
> `/post/:site/:post/revisions`
will do :) 

> Also, it doesn't have to be done in this PR, but it's worth thinking about whether the approach we're using will handle revisions for post types other than post (page and custom post types).
It seems for pages, there will be some changes needed when querying as the URI changes to use `/pages`. But - as you said - considering this one PR is already XL, I will probably tackle that in a subsequent PR.